customize-robots.Rmd

```diff
@@ -90,53 +116,84 @@ These previews do NOT incorporate any changes influenced by any changes to the D
 
 In the `config_automation.yml` file it is set by:
 ```
-spell-check: yes
+spell-check: true
 ```
 Github Actions will automatically run a spell check on all Rmds and mds whenever a pull request to the `main` branch is filed.
 
-It will fail if there are more than 2 spelling errors and you'll need to resolve those before being able to merge your pull request. Errors will be printed out on a GitHub comment on your pull request.
+The error threshold option controls how many errors are allowed before the check will fail.
 
-![](https://raw.githubusercontent.com/ottrproject/OTTR_Template/main/resources/screenshots/spell_check_comment.png)
+```
+spell_error_min: 0
+```
+
+You'll need to resolve those before being able to merge your pull request. Errors are displayed in much the same way as PR URL check errors with a summary of how many errors are found. Click on the output comment on your pull request that says "Click here to download detailed error reports". This will download a zip file containing a list of the spelling errors it found.
```

Comment:
```suggestion
You'll need to resolve those before being able to merge your pull request. Errors are displayed in much the same way as URL check errors with a summary of how many errors are found. Click on the output comment on your pull request that says "Click here to download detailed error reports". This will download a zip file containing a list of the spelling errors it found.
```